statsfs/otelstats/otel_metrics.go

```diff
@@ -0,0 +1,80 @@
+package otelstats
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"strconv"
+	"strings"
+
+	"go.opentelemetry.io/otel/api/global"
+	"go.opentelemetry.io/otel/api/kv"
+	"go.opentelemetry.io/otel/api/metric"
+)
+
+// InitOtelPipeline initializes an OpenTelemetry pipeline
+// that crawls a user defined path and exports all the available
+// stats to a backend of choice (gcp, stdout, prometheus)
+func InitOtelPipeline(exporterName, statsfsPath string) {
+	exporter, err := InitExporter(exporterName)
+	if err != nil {
+		log.Panicf("Failed to initialize exporter %v: %v\n", exporterName, err)
+	}
+	if exporter != nil {
+		defer exporter.Stop()
+	}
+
+	err = createOtelMetricsForStatsfs(statsfsPath)
+	if err != nil {
+		log.Panic(err)
+	}
+
+	for {
+	}
```

Comment:
This is something we definitely do not want :) It will eat up all the CPUs.

Follow some suggestions in https://blog.sgmansfield.com/2016/06/how-to-block-forever-in-go/ instead. Please make sure that the one you choose can be terminated by Ctrl-C.